src/grammar.coffee

```diff
@@ -154,7 +154,11 @@ URL_VALIDATOR = /// ^
             )
         | git/ (
               refs
-            | refs / heads (/[^/]+)?
+            | refs / (
+                .+
+                | heads (/[^/]+)?
```

Comment:
Would `| refs (/[^/]+)*` on line 157 describe all of the options better (or enumerate `pulls | heads | tags`)? I have tried to avoid using `.+` in the regexp